js/package.json

```diff
@@ -25,7 +25,7 @@
     "eslint": "^7.14.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-prettier": "^3.2.0",
-    "mongodb-json-schema-to-typescript": "^9.3.0",
+    "json-schema-to-typescript": "^10.1.3",
```

Comment:
Pour info: `mongodb-json-schema-to-typescript` était un _fork_ de `json-schema-to-typescript` qui s'est avéré trop compliqué à modifier, suite à des mises à jour récentes.